dds/src/Conversion.cpp

```diff
@@ -18,6 +18,35 @@
 #include "Conversion.hpp"
 #include <sstream>
 #include <soss/utilities.hpp>
+#include <fastrtps/types/MemberDescriptor.h>
+
+using eprosima::fastrtps::types::ResponseCode;
+using eprosima::fastrtps::types::MemberDescriptor;
+using eprosima::fastrtps::types::MemberId;
+
+// Due to a problem with Fast-RTPS library, which by now does not export the "GetDescriptor" funciton for windows,
+// this workaround is needed in order to access the protected member "mDescriptors".
+// This problem will be solved in future fixes, but for now this workaround is the only way of introspecting
+// dynamic types in windows.
+class DynamicDataSOSS : public eprosima::fastrtps::types::DynamicData
+{
+public:
+    ResponseCode GetDescriptorSOSS(MemberDescriptor& value, MemberId id) const
+    {
+        auto it = mDescriptors.find(id);
+        if (it != mDescriptors.end())
+        {
+            value.CopyFrom(it->second);
+            return ResponseCode::RETCODE_OK;
```

Comment:
Instead of return the OK by yourself, let the CopyFrom function returns the `ResponseCode`

Reply:
done